### PR TITLE
feat: add deadline selection for termins

### PR DESCRIPTION
**Pull request by aestuans**
Add the option to be notified only if an appointment is available before a specified deadline.
The user is prompted to specify a the number of days `n`, and they will only be notified if an appointment is available within the next `n` days.
This is useful because appointments are frequently available in several months in the future, while we are only interested in more immediate appointments and don't want to be notified otherwise. See #10.

### CODE REVIEW

File: job_storage.py [hunk 1]
```diff
@@ -41,8 +41,11 @@ def add_subscription(update, context, interval):
 
     buro = context.user_data['buro']
     termin = context.user_data['termin_type']
+    deadline = context.user_data['deadline']
```
Comment (okainov) [inline job_storage.py:44]: Will it work with existing subscriptions?

File: printers.py [hunk 2]
```diff
@@ -29,16 +29,18 @@ def print_subscription_status(update, context):
...
-            'Current subscription details:\n\n - Department: %s \n - Type: %s \n - Interval: %s \n - Until: %s \n' % (
-                department.get_name(), subscription.kwargs['termin'], subscription.trigger.interval, date_object))
+            'Current subscription details:\n\n - Department: %s \n - Type: %s \n - Interval: %s \n - Until: %s \n - Deadline: %s \n' % (
+                department.get_name(), subscription.kwargs['termin'], subscription.trigger.interval,
+                subscription_limit_date_time, deadline))
         print_unsubscribe_button(chat_id)
 
 
-def notify_about_termins(chat_id, buro, termin, created_at):
+def notify_about_termins(chat_id, buro, termin, created_at, deadline):
```
Comment (okainov) [inline printers.py]: Please make deadline optional to keep existing subscriptions working

**Review by okainov — CHANGES_REQUESTED** [review]
Please fix handling existing subscriptions, the rest looks fine, thanks for PR!

File: printers.py (hunk 2 above)
Comment (aestuans) [reply printers.py]: Done :+1:

**Comment by okainov** [comment]
Merged with #70, sorry for taking so long :(

Thanks for contribution!

**Pull request closed without merging**